src/GreeterWidget.vala

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2011-2015 Wingpanel Developers (http://launchpad.net/wingpanel)
+ * Copyright (c) 2011-2018 elementary, Inc. (https://elementary.io)
```

Comment:
Might as well use the current year :smile: 

```suggestion
 * Copyright (c) 2011–2019 elementary, Inc. (https://elementary.io)
```

Reply:
LOL, copy-paste mistake, thank you for noticing.